src/nodes.coffee

```diff
@@ -1186,7 +1192,7 @@ exports.Obj = class Obj extends Base
     for prop in @properties
       prop = prop.value if prop instanceof Assign and prop.context is 'object'
       prop = prop.unwrapAll()
-      prop.eachName iterator
+      prop.eachName iterator if prop.eachName?
```

Comment:
I'd be inclined to leave the existence check off - the reason being that anything that's `isAssignable` should support it, and we've already checked assignability at this point.